test/main.tf

```diff
@@ -0,0 +1,69 @@
+/*
+resource "keycloak_client" "client_test" {
+  realm     = "Jenkins"
+  client_id = "test"
+  redirect_uris = ["http://127.0.0.1:8080/"]
+  service_accounts_enabled = true
+}
+*/
+
+resource "keycloak_group" "group1" {
+  name       = "group1"
+  realm      = "Jenkins"
```

Comment:
Did you consider referencing terraform resource ids or similar here, rather than static realm names?